packages/components/Notification/src/__tests__/__snapshots__/Notification.test.tsx.snap

```diff
@@ -211,14 +172,7 @@ exports[`Notification component tests Notification default 1`] = `
         numberOfLines={0}
         style={
           Object {
-            "color": Object {
-              "dynamic": Object {
-                "dark": "#77B7F7",
-                "highContrastDark": undefined,
-                "highContrastLight": undefined,
-                "light": "#0F6CBD",
-              },
-            },
+            "color": "#0f548c",
```

Comment:
Total nit - does casing matter here for consistency? Upper vs lower case for hex values?

Reply:
This is a generated file. I think the difference is the hardcoded colors we had uppercased, while the tokens (coming from the token pipeline) are lowercased. Either way, that should not affect the final color.